content/pt/docs/languages/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: APIs e SDKs de Linguagens
+description:
+  A instrumentação de código do OpenTelemetry é suportada para muitas linguagens
+  populares de programação.
+weight: 250
+redirects: [{ from: /docs/instrumentation/*, to: ':splat' }]
+default_lang_commit: b3a087c468094eb144ab3114aff06838c8fb06ca
+---
+
+A [instrumentação][] de código do OpenTelemetry é suportada para as linguagens
+listadas na tabela de [Estado e Lançamentos](#estado-e-lançamentos) abaixo.
+Implementações não oficiais para [outras linguagens](/docs/languages/other)
+também estão disponíveis. Você pode encontrá-las no
+[registro](/ecosystem/registry/).
+
+Para Go, .NET, PHP, Python, Java e JavaScript, você pode usar
+[soluções de instrumentação automática](/docs/zero-code) para instrumentar suas
+aplicações sem alterações de código.
+
+Se você estiver usando Kubernetes, você pode usar o [OpenTelemetry Operator para
+Kubernetes][otel-op] para [injetar soluções de implementação
+automática][implementação automática] em suas aplicações.
+
+## Estado e Lançamentos
```

Comment:
```suggestion
## Estado e Lançamentos {#original-english-id}
```
See above and https://opentelemetry.io/docs/contributing/localization/#heading-anchors